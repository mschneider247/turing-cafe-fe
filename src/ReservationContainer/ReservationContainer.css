.reservations {
  display: flex;
  flex-wrap: wrap;
}