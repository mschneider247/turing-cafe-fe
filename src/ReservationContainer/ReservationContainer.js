import React from 'react';
import './ReservationContainer.css'
import Reservation from '../Reservation/Reservation'

const ReservationContainer = (props) => {

  console.log("Inside ReservationContainer props.reservations:::", props.reservations)

  let reservations = props.reservations.map(reservation => {
    return <Reservation 
      key={reservation.id}
      id={reservation.id}
      name={reservation.name}
      date={reservation.date}
      time={reservation.time}
      numGuests={reservation.number}
      removeReservation={props.removeReservation}
      />
  })

  return (
    <section className="reservations">
      {reservations}
    </section>
  )
}

export default ReservationContainer;