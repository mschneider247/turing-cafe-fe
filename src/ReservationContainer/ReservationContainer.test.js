import React from 'react';
import { shallow } from 'enzyme';
import ReservationContainer from './ReservationContainer';

describe('Form', () => {

  it('Should match the snapshot', () => {
    const mockReservations = [{id: 1, name: 'steve'}, {id: 2, name: 'bob'}, {id: 3, name: 'nancy'}]
    const wrapper = shallow(<ReservationContainer reservations={mockReservations}/>)
    expect(wrapper).toMatchSnapshot();
  })
})