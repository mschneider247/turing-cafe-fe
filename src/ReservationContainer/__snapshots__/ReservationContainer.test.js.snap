// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Should match the snapshot 1`] = `
<section
  className="reservations"
>
  <Reservation
    id={1}
    key="1"
    name="steve"
  />
  <Reservation
    id={2}
    key="2"
    name="bob"
  />
  <Reservation
    id={3}
    key="3"
    name="nancy"
  />
</section>
`;
